README.md

```diff
@@ -1,36 +1,19 @@
 # Krun
 
-Krun is a framework for running software benchmarking experiments.
+Krun is a framework for running high-quality software benchmarking experiments.
+Krun experiments consist of a configuration file, a carefully configured
+benchmarking machine, and the Krun program itself.
 
-**Krun uses sudo to elevate privileges! Please read these instructions in
-full.**
 
-The `examples/` directory contains a simple experiment using Krun.
-This is a good starting point for setting up your own Krun configuration.
+## Step 1: Initial installation
 
-The example here contains two benchmark programs (*nbody* and *dummy*),
-executed on two VMs (*cPython* and native code with no VM). There is a separate
-example with Java (using *JVM* such as HotSpot), which requires you to set up
-some environment variables before you compile Krun.
-Each benchmark is run for 5 iterations on the same VM, then the VM is
-restarted and the benchmark is re-run for another 5 iterations.
-We say that the experiment runs 2 *process executions* and 5 *in-process iterations* of
-each benchmark.
-
-This configuration can be found in the file `examples/example.krun`. The
-example with Java can be found in `examples/java.krun`.
-
-## Step 1: prepare the benchmarking machine
-
-Krun currently only runs on Unix-like environments.
-To run this example experiment, you need superuser rights to the machine you are
-using, e.g. on Linux you should be able to run `sudo`.
+Krun currently only runs on Unix-like environments: it has been tested
```

Comment:
It works on Ubuntu (somewhat) IIRC.

Reply:
So do we want to support their kernel customisations too? I'd argue that ubuntu is a desktop oriented distro not well suited to benchmarking anyway.

Reply:
Sarah has run things on Ubuntu; Tom on Arch. I don't think we can, or should, say "Debian or nothing" (though we can say "Debian is best for this task") as we'll put people off getting *some* improvements.

Reply:
It kinda works on Ubuntu because it is almost Debian, but we won't be making Linux kernel versions for Ubuntu.

I'd be amazed if Arch worked. Are you sure it worked? IIRC there's a check for `/etc/debian_version`:

https://github.com/softdevteam/krun/blob/cf1ee595c2b07e19790f750f9a650422a815e0ec/krun/platform.py#L1410-L1420

Reply:
Ah, I think Tom used Debian on his laptop so I was wrong that it works on Arch. OK, let me adjust things accordingly.